source/Marquee.js

```diff
@@ -232,6 +232,17 @@
 		* @method
 		* @private
 		*/
+		destroy: enyo.inherit(function (sup) {
+			return function () {
+				moon.Marquee.setMarqueeOnHoverControl(null);
```

Comment:
Hi @clinuz ,
I'm not sure about completely understood or not on second comment. 
The sentence structure is not familiar to me @.@
Anyway, I think I can add a condition check before setting back to null on destroy.
Do you want to add handling on a leave event also?